src/main/java/Duke.java

```diff
@@ -1,10 +1,69 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+
 public class Duke {
     public static void main(String[] args) {
         String logo = " ____        _        \n"
```

Comment:
Perhaps you want to consider making the logo a FINAL variable? since it wont change. Other than that your code looks good